pipeline_steps/kibana.groovy

```diff
@@ -62,7 +62,9 @@ def kibana_tests(vm=null){
         export PATH=\$PATH:./phantomjs-2.1.1-linux-x86_64/bin
         # Remove any existing screenshots from old runs
         rm -f *.png
-        python conf-gen.py --secure --password-file /etc/openstack_deploy/user_rpco_secrets.yml \
+
+        export PASSWORD=\$(grep -Ir kibana_password /etc/openstack_deploy/ | head -1 | sed 's/.*\\: //')
```

Comment:
For liberty to mitaka upgrades, I was seeing the kibana password in user_extras_secrets.yml and user_rpco_secrets.yml, which were identical files. The first one is what liberty uses and the second file is standard for mitaka and newton. I'll look into why there are duplicate files.